nixos/doc/manual/release-notes/rl-1803.xml

```diff
@@ -231,6 +231,14 @@ following incompatible changes:</para>
       <literal>Option "XkbRules" "base"</literal> may result in broken keyboard layout.
     </para>
   </listitem>
+  <listitem>
+    <para>
+      The <literal>attic</literal> packaged was removed. A maintained fork called
+      <link xlink:href="https://www.borgbackup.org/">Borg</link> should be used instead.
```

Comment:
Would it make sense to also add `attic = borg;` in `pkgs/top-level/aliases.nix`?

Reply:
I don't think so. The executables have different names.

Reply:
But I just spotted a typo: packaged instead of package. Corrected that.